pkg/security/security.go

```diff
@@ -58,6 +58,11 @@ var (
 	TokenAudiences = strings.Split(env.RegisterStringVar("TOKEN_AUDIENCES", "istio-ca",
 		"A list of comma separated audiences to check in the JWT token before issuing a certificate. "+
 			"The token is accepted if it matches with one of the audiences").Get(), ",")
+
+	XDSTokenType = env.RegisterStringVar("XDS_TOKEN_TYPE", "Bearer",
```

Comment:
nit: this seems to be used for CA as well, but API has "XDS"

Reply:
Ack, will do a follow-up - there are other related changes and infra that is using this.